src/bin.ts

```diff
@@ -51,10 +52,18 @@ async function run(argv) {
   if (language === 'typescript') {
     require('ts-node').register()
   }
-  const generatorInstance =
-    language === 'typescript'
-      ? new TypescriptGenerator(args)
-      : new Generator(args)
+  let generatorInstance
```

Comment:
hi 🙌 maybe it would be a good idea to move this logic outside to have e.g. :

```js
const getGeneratorConstructor = language => {
  switch (language) {
    case 'typescript':
      return TypescriptGenerator;
    case 'flow':
      return FlowGenerator;
    default:
      return Generator;
  }
};
```
and then 
```js
const generatorInstance = new (getGeneratorConstructor(language))();
```